src/components/menu/menu.xml

```diff
@@ -16,10 +16,10 @@
             <div
               t-att-title="getName(menuItem)"
               t-att-data-name="menuItem.id"
-              t-on-click="(ev) => this.onClickMenu(menuItem, menuItem_index, ev)"
-              t-on-mouseover="(ev) => this.onMouseOver(menuItem, menuItem_index, ev)"
-              class="o-menu-item"
-              t-att-class="{ 'o-menu-root': isMenuRoot, 'disabled': !isMenuEnabled, 'o-menu-item-active': isParentMenu(subMenu, menuItem)}"
+              t-on-click="(ev) => this.onClickMenu(menuItem, ev)"
+              t-on-mouseover="(ev) => this.onMouseOver(menuItem, ev)"
+              class="o-menu-item d-flex justify-content-between align-items-center"
+              t-att-class="{'disabled': !isMenuEnabled, 'o-menu-item-active': isParentMenu(subMenu, menuItem)}"
```

Comment:
With your PR, we can have 2 menu item active in the same menu (and colored gray): the one that have a submenu open, and the one that is hovered. There should only be one active at all time.